packages/shared/lib/core/wallet/utils/convertBech32ToEd25519.ts

```diff
@@ -0,0 +1,8 @@
+import { networkHrp } from '@core/network'
+import { Bech32Helper } from '@lib/bech32Helper'
+import { Converter } from '@lib/converter'
+import { get } from 'svelte/store'
+
+export function convertBech32ToEd25519(bech32: string): string {
```

Comment:
Change name to be inclusive of all address types. And then also to make this the reverse of the convertHexAddressToBech32 we should be adding the hex prefic '0x'